gradle/verification-metadata.xml

```diff
@@ -0,0 +1,4267 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<verification-metadata xmlns="https://schema.gradle.org/dependency-verification" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="https://schema.gradle.org/dependency-verification https://schema.gradle.org/dependency-verification/dependency-verification-1.0.xsd">
+   <configuration>
+      <verify-metadata>true</verify-metadata>
```

Comment:
new way to secure/validate dependencies https://docs.gradle.org/current/userguide/dependency_verification.html#sub:enabling-verification